src/app/pages/vm/vm-cards/vm-cards.component.ts

```diff
@@ -523,12 +524,14 @@ export class VmCardsComponent implements OnInit,OnDestroy {
     }
      else {
       if(poweroff){
-         this.off_text =  `Do you wish to Power Off ${vm.name} ?`;
+         this.off_text =  `Power off to ${vm.name} ?`;
```

Comment:
Please see my earlier comment on this, which included the suggested code that addresses the problems this still has, like the extraneous "to", the lack of delimiters around the vm name, and the extraneous space before the question mark.

Reply:
what `delimiters` do you want ?